SUIApp/SUIApp/ViewModel/FilterViewModel.swift

```diff
@@ -0,0 +1,50 @@
+// FilterViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// ViewModel для экрана с фильтрами
+final class FilterViewModel: ObservableObject {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let bedImageNameText = "bed"
+        static let buffetImageNameText = "buffet"
+        static let chairImageNameText = "chair"
+        static let sofaImageNameText = "sofa"
+        static let stolImageNameText = "stol"
+        static let sliderNumeberRange: ClosedRange<Float> = 1...1000
+    }
+
+    // MARK: - Public Property
+
+    @Published var furnitureImageNames = [
```

Comment:
```suggestion
    var furnitureImageNames = [
```
сто проц не нужен паблиш